maplibre_gl_example/lib/animate_camera.dart

```diff
@@ -73,6 +74,27 @@ class AnimateCameraState extends State<AnimateCamera> {
                   },
                   child: const Text('newCameraPosition'),
                 ),
+                TextButton(
+                  onPressed: () {
+                    mapController
+                        .easeCamera(
+                          CameraUpdate.newCameraPosition(
+                            const CameraPosition(
+                              bearing: 270.0,
+                              target: LatLng(46.233487, 14.363610),
+                              tilt: 30.0,
+                              zoom: 17.0,
+                            ),
+                          ),
+                          const Duration(milliseconds: 500),
+                        )
+                        .then(
+                          (result) => debugPrint(
+                              "mapController.easeCamera() returned $result"),
+                        );
+                  },
+                  child: const Text('easeCamera'),
+                ),
```

Comment:
`easeCamera` isn't available on web, add a check to don't show this example when the device is web
```suggestion
                if(!kIsWeb)
                  TextButton(
                    onPressed: () {
                      mapController
                          .easeCamera(
                            CameraUpdate.newCameraPosition(
                              const CameraPosition(
                                bearing: 270.0,
                                target: LatLng(46.233487, 14.363610),
                                tilt: 30.0,
                                zoom: 17.0,
                              ),
                            ),
                            const Duration(milliseconds: 500),
                          )
                          .then(
                            (result) => debugPrint(
                                "mapController.easeCamera() returned $result"),
                          );
                    },
                    child: const Text('easeCamera'),
                  ),
```